submissions/bilkisuTace/phpDatabases/setCookies.php

```diff
@@ -0,0 +1,17 @@
+<?php
+include('db_con.php');
+    function setData(){
```

Comment:
A better approach would have been

- let this function `return` `$data`
- Remove the `if` statement you are doing here. Move the check to line 3 of your `customers.php` file
- On your `customers.php` file you should then do this 
```
if(!isset($_COOKIE['customers'])) { 
     $results = setData();
} else {
      $results = json_decode($_COOKIE['customers'], true);
}